pytest_splinter/hooks.py

```diff
@@ -0,0 +1,9 @@
+from pluggy import HookspecMarker
+
+hookspec = HookspecMarker("pytest")
+
+
+@hookspec(firstresult=True)
+def pytest_splinter_screenshot_dir(config):
```

Comment:
still, please document the hook in the readme

Reply:
Done